packages/styling/src/styles/theme.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as theme from './theme';
+import { ITheme } from '../interfaces/index';
+
+describe('registerOnThemeChangeCallback', () => {
+  let counter = 0;
+  let callback = (t: ITheme) => {
```

Comment:
Can you use jest.fn() instead of making a custom callback function?

Reply:
Wow that's useful.
I did that, but get this error now trying to build:

[@uifabric/styling 7:33:45 PM] Starting: ts
error TS2688: Cannot find type definition file for 'jest'.

[@uifabric/styling 7:33:47 PM] Error: ts (1.835s): Error: Command failed: node E:\git\office-ui-fabric-react\scripts\node_modules\typescript
\lib\tsc -outDir lib -t es5 -m commonjs --pretty

Do you know how to fix that?